schemas/hygrothermalData.json

```diff
@@ -1,37 +1,183 @@
+          "$ref": "number.json#/$defs/kilogramPerSquareMeterAndSquareRootSeconds",
+          "description": "short description"
+        },
+        "moistureDependentConductivitySupplement": {
+          "$ref": "number.json#/$defs/percentPerMassPercent",
+          "description": "short description"
+        },
+        "temperatureDependentConductivitySupplement": {
+          "$ref": "number.json#/$defs/wattPerMeterSquareKelvin",
+          "description": "short description"
+        },
+        "typicalBuildInMoisture": {
+          "$ref": "number.json#/$defs/kilogramPerCubicMeter",
+          "description": "short description"
+        },
+        "moistureStorageFunction": {
+          "type": "array",
+          "items": {
+            "type": "object",
+            "properties": {
+              "relativeHumidityAbsolut": {
+                "$ref": "number.json#/$defs/percentDecimalBetweenZeroAndOne",
+                "description": "short description"
+              },
+              "waterContent": {
+                "$ref": "number.json#/$defs/kilogramPerCubicMeter",
+                "description": "short description"
+              }
             }
-          },
-          "additionalProperties": false,
-          "minProperties": 1,
-          "required": ["xmlData"]
+          }
+        },
+        "moistureStorageFunctionGenerated": {
+          "type": "boolean"
+        },
+        "liquidTransportCoefficientSuction": {
+          "type": "array",
+          "items": {
+            "type": "object",
+            "properties": {
+              "waterContent": {
+                "$ref": "number.json#/$defs/kilogramPerCubicMeter",
+                "description": "short description"
+              },
+              "liquidTransportCoefficient": {
+                "$ref": "number.json#/$defs/squareMeterPerSecond",
+                "description": "short description"
+              }
+            }
+          }
+        },
+        "liquidTransportCoefficientSuctionGenerated": {
+          "type": "boolean"
+        },
+        "liquidTransportCoefficientRedistribution": {
+          "type": "array",
+          "items": {
+            "type": "object",
+            "properties": {
+              "waterContent": {
+                "$ref": "number.json#/$defs/kilogramPerCubicMeter",
+                "description": "short description"
+              },
+              "liquidTransportCoefficient": {
+                "$ref": "number.json#/$defs/squareMeterPerSecond",
+                "description": "short description"
+              }
+            }
+          }
+        },
+        "liquidTransportCoefficientRedistributionGenerated": {
+          "type": "boolean"
+        },
+        "thermalConductivityMoistureDependent": {
+          "type": "array",
+          "items": {
+            "type": "object",
+            "properties": {
+              "waterContent": {
+                "$ref": "number.json#/$defs/squareMeterPerSecond",
```

Comment:
Please double check all units. The waterContent is probably not in squareMeterPerSecond.

Reply:
yes, it ius kilogramPerCubicMeter. checked units.